apps/ui/src/networks/offchain/actions.ts

```diff
@@ -56,11 +60,20 @@ type ReadOnlyExecutionPlugin = {
   safes: ReadOnlyExecutionSafe[];
 };
 
-const CONFIGS: Record<number, OffchainNetworkConfig> = {
+const CONFIGS: Record<number, OffchainNetworkEthereumConfig> = {
   1: offchainMainnet,
   5: offchainGoerli
 };
 
+const STARKNET_CONFIGS: Record<number, OffchainNetworkStarknetConfig> = {
+  1: offchainStarknetMainnet,
+  11155111: offchainStarknetSepolia
+};
+
+const STARKNET_CHAIN_IDS = Object.entries(
+  starknetConstants.StarknetChainId
+).flat();
```

Comment:
I think this will return`['SN_MAIN', '0x534e5f4d41494e', 'SN_SEPOLIA', '0x534e5f5345504f4c4941']`

```suggestion
const STARKNET_CHAIN_IDS = Object.values(
  starknetConstants.StarknetChainId
).flat();
```

Reply:
This is intentional:

https://github.com/snapshot-labs/sx-monorepo/pull/1588/files#diff-c35aa4cf776b150ef8a63ff4cea8023dab729f65529e8d4d89d47a37d28d66ceR419

Reply:
Can we add comment for that constant here? Before someone comes back to it in couple months and "refactors" it 😆